src/sql/gazetteer_geometry_views.sql

```diff
@@ -23,16 +23,20 @@ SELECT
 from
     gazetteer.feature;
 
-CREATE RULE feature_ref_point_ins AS ON INSERT TO feature_ref_point
+CREATE OR REPLACE RULE feature_ref_point_ins AS ON INSERT TO feature_ref_point
 DO INSTEAD
    (
    INSERT INTO feature (feat_type, status, description, ref_point )
    VALUES (NEW.feat_type, 'CURR', '', NEW.ref_point );
    INSERT INTO name (feat_id, name, status )
    VALUES (lastval(), NEW.name, 'UNEW' )
-   );
+   RETURNING
+     feat_id,
+     gaz_preferredName(feat_id) as name,
+     (SELECT f.feat_type FROM feature f WHERE f.feat_id = feat_id),
+     (SELECT f.ref_point FROM feature f WHERE f.feat_id  = feat_id));
```

Comment:
Extra space here `  =`

Reply:
thanks. Other instances found and removed also